SwiftCompilerSources/Sources/AST/DiagnosticEngine.swift

```diff
@@ -14,8 +14,6 @@ import ASTBridging
 
 import Basic
 
-public typealias DiagID = BridgedDiagID
-
```

Comment:
can we do `public typealias DiagID = swift.DiagID` to avoid adding the `swift.` prefix everywhere?

Reply:
Yeah, the underlying C++ implementation shouldn't leak into the API. The APIs should look like if the whole compiler would be implemented in swift - which might be the case in the far future :-)